modules/lib/api.json

```diff
@@ -371,7 +371,7 @@
                 ],
                 "operationId": "capi:upload",
                 "x-constraints": {
-                    "group": "tei"
+                    "groups": "tei"
```

Comment:
See https://github.com/eeditiones/roaster/blob/master/content/auth.xql#L67-L69. The key `groups` is correct, but it appears that the value needs to be expressed as an array.

Reply:
Given the code I linked to, I am surprised that the call to `$user?groups?*` doesn't throw an error. The following query:

```xquery
map {"a": 1}?a?*
``` 

... throws:

> err:XPTY0004 expression to the left of a lookup operator needs to be a sequence of maps or arrays [at line 1, column 15]

I wonder if some try-catch expression is swallowing the error?

Reply:
i made it an array